install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2892,12 +2891,7 @@ operator:
     kubernetes.io/os: linux
   # -- Node tolerations for cilium-operator scheduling to nodes with taints
   # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
-  tolerations:
-    - operator: Exists
-      # - key: "key"
-      #   operator: "Equal|Exists"
-      #   value: "value"
-      #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+  tolerations: ~
```

Comment:
Just FYI this looks like a breaking change and I'm not sure we can adopt this without breaking Cilium installation or upgrade for existing users.

Reply:
To deploy Cilium initially, the first thing to do is deploy cilium-operator, before even setting up a CNI. This means that almost all possible taints should be tolerated. Evidently for this use case there needs to be at least one taint that is not tolerated, specifically some taint used when draining the nodes. 

I'm not familiar with what possible taints might exist in arbitrary environments or how to better express this notion of "ignore everything except draining" in the Helm charts, but perhaps @cilium/sig-k8s or @cilium/helm reviewers could provide some insights.

Reply:
@squeed You mentioned that there should exist a subset of tolerated taints in https://github.com/cilium/cilium/issues/28549#issuecomment-1761348690 - any suggestions what taints in particular we should tolerate?